jacobian/SWMHD_example.jl

```diff
@@ -72,7 +86,25 @@ simulation.output_writers[:fields] = JLD2OutputWriter(model, (; u, v, A = model.
                                                       overwrite_existing = true)
 
 @info "Running with Δt = $(prettytime(simulation.Δt))"
+sim_start_time = time_ns()*1e-9
 run!(simulation)
+sim_end_time = time_ns()*1e-9
+sim_time = prettytime(sim_end_time - sim_start_time)
+@info "Simulation took $(sim_time) to finish running."
+
+u, v, h = simulation.model.solution
```

Comment:
again, i'm not sure why we do all of this again?

Reply:
Hmm do you think it would be sufficient to call compute!(...) again to get the final energy? I'm not sure if that's sufficient -  I was redefining the variables because we want these variables at the most recent time step (the final time step) so we can compute the final total energy